atlp-rwanda/Phantom-Backend-Elites
# Bg-fix-develop

### What does this PR do?
This PR is aimed at fixing different bugs in develop. The bugs include the following: 
- Production build scripts
- User's controller
- Assign Drivers to buses controller

### How should this be manually tested?
- clone the repository,then run `npm i` to install all depandencies.
- start the server with `npm run dev` which will also create the tables in your database

**N.B:  Use the following endpoints to test. Note that the port(9000) can be changed to any port within your `.env`**
- CRUD for users: `localhost:9000/api/v1/users`
- Assign Drivers to buses: `http://localhost:9000/api/v1/drivers/assign-bus`
